testing/tests/DevExpress.ui.widgets/menu.tests.js

```diff
@@ -1885,6 +1886,36 @@ QUnit.test("Adaptivity should be available for horizontal orientation only", fun
     assert.equal($adaptiveContainer.length, 0, "adaptiveContainer was not rendered");
 });
 
+QUnit.test("maxHeight should be 90% of maximum of top or bottom offsets when height of overlay content more windows height", function(assert) {
+    var getData = function(count) {
+        var result = [];
+        for(var i = 0; i < count; i++) {
+            result.push({ text: "item_" + i });
+        }
+        return result;
+    };
+
+    new Menu(this.$element, {
+        items: getData(10),
+        adaptivityEnabled: true
+    });
+
+    var scrollTop = sinon.stub(renderer.fn, "scrollTop").returns(100),
+        windowHeight = sinon.stub(renderer.fn, "innerHeight").returns(700),
+        offset = sinon.stub(renderer.fn, "offset").returns({ left: 0, top: 200 });
+
+    try {
+        var overlay = $(".dx-overlay").dxOverlay("instance"),
```

Comment:
Can we use `menu.content().closest(".dx-overlay")` to prevent finding of the overlay from another test?